.travis.yml

```diff
@@ -5,30 +5,48 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
-  - nightly
+
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - $HOME/.cache/pip
+    - vendor
```

Comment:
What does this do ? Not sure we want to cache that (especially if using `--prefer-lowest`)